Product OS docs refresh (#7800)
PostHog/posthog.com: contents/docs/data/actions.mdx

---

## What is an action?

Actions are a way of combining several related events into one, which you can then analyze in [insights](/docs/product-analytics/insights) and [dashboards](/docs/product-analytics/dashboards) as if it were a single event.

Actions can include autocapture events, custom events, or pageviews, and support filters for:

- Event properties (e.g. UTM tags)
- Person properties (e.g. location, operating system)
- Feature flags (e.g. beta test groups and feature toggles)
- Cohorts (e.g. groups of users with related properties)
- Autocapture elements (e.g. text and CSS selectors)

You can also use actions to "rename" events by creating an action based on a single event and giving it easy to understand name.

## How are actions useful?

Actions are great for analzying related events as if they were one.

In PostHog, for example, we track custom events like `insight created`, `insight analyzed`, and `dashboard created`. These are all different ways to interacting with product analytics.

We then combine these and similar events into an action called `Product Analytics Interactions`, which gives us an aggregate view of how much people are using product analytics.

> Actions are also useful for tracking conversions on your website. Our website homepage, for example, has several call to actions (CTAs) to signup to PostHog. We combine all these events into a single `Homepage CTA` action, so we can track all clicks from the signup page in a trends insight.

Reviewer: This is probably not true, but it is a good use case!
```suggestion
Actions are also useful for tracking conversions on your website. Our website homepage, for example, has several call to actions (CTAs) to signup to PostHog. We combine all these events into a single `Homepage CTA` action, so we can track all clicks from the signup page in a trends insight and run A/B tests with this action as a goal.
```

## How to create an action

You can create new actions by either: 

> 1. Clicking the "New action" button in the 'Actions' tab in [Data Management](https://app.posthog.com/data-management/events).
> 
> 2. Using the [PostHog Toolbar](/docs/toolbar) to tag the elements that are relevant – see our step-by-step tutorial on [creating actions using the toolbar](https://posthog.com/tutorials/how-to-capture-events-the-easy-way).

Reviewer: ```suggestion
1. Clicking the "New action" button in the "Actions" tab in [Data Management](https://us.posthog.com/data-management/actions).

2. Using the [PostHog Toolbar](/docs/toolbar) to tag the elements that are relevant – see our step-by-step tutorial on [creating actions using the toolbar](/tutorials/how-to-capture-events-the-easy-way).
```

Actions are retroactive, meaning they also apply to past events, not only events that happen after the action was created.

You can create actions using events from three sources:

### 1. Autocapture

[Autocapture](/docs/product-analytics/autocapture) actions are based on frontend elements from your website, such as a button or an input. 

> The easiest way to select them is [using the toolbar](https://posthog.com/tutorials/how-to-capture-events-the-easy-way), but you can also set them manually by providing something to identify the element, like a selector.

Reviewer: ```suggestion
The easiest way to select them is [using the toolbar](/tutorials/how-to-capture-events-the-easy-way), but you can also set them manually by providing something to identify the element, like a selector.
```
